src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements GetFigureArea {
+    private double radius = 5;
+
+    @Override
+    public String toString() {

```

Comment:
According to the task description, you should not override the `toString()` method to 'draw' the figure. Instead, create a separate method to print out all information about the figure using `System.out.println()`.